system_baseline/models.py

```diff
@@ -118,7 +118,11 @@ class SystemBaselineMappedSystem(db.Model):
 
     @classmethod
     def delete_by_system_ids(cls, system_ids, account_number, org_id):
-        query = cls.query.filter(cls.system_id.in_(system_ids), cls.account == account_number)
+        if org_id:
+            query = cls.query.filter(cls.system_id.in_(system_ids), cls.org_id == org_id)
```

Comment:
Please fixup this commit to commit `Adds org-id into code where account is used`

Reply:
Done